index.html

```diff
@@ -0,0 +1,559 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="normalize.css">
+    <link href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" rel="stylesheet">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="prefixfree.min.js">
+    <title>Responsive-Design</title>
+</head>
+<body>
+    <header>
+        <div class="header-1">
+            <ul class="left-section items-left">
+                <li>  <a href="#"> News</a></li>
+                <li><a href="#">Events</a></li>
+                <li> <a href="#">Business</a></li>
+                <li><a href="#">AMAs</a></li>
+                <li> <a href="#">Spaces</a></li>
+                <li><a href="#">Terms & Conditions</a></li>
+            </ul>
+            <ul class="right-section items-right">
+                <li><a href="#">About</a></li>
+                <li><a href="#">Team</a></li>
+                <li><a href="#">Advertise</a></li>
+                <li><a href="#">Jobs</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </div>
+        <nav class="nav-bar">
+            <ul class="nav-bar-ul">
+                <li><img class="tnw-img" src="images/tnw.png" alt="tnw logo"></li>
+                <li><a href="#">latest</a></li>
+                <li><a href="#">hard fork</a></li>
+                <li><a href="#">plugged</a></li>
+                <li><a href="#">fundamentals</a></li>
+                <li><a href="#">readme</a></li>
+                <li><a href="#">growth quarters</a></li>
+                <li><a href="#">neurals</a></li>
+            </ul>
+            <ul class="search-bar-ul">
+                <li class="search-icon right-icon"><a href="#"><i class="fas fa-search"></i></a></li>
+                <li><img class="tnw-2020" alt="TNW Conference" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/misc/event-bl-tnw2020.svg"></li>
+                <li class="search-bar-txt">kicks off in</li> 
+                <li class="days">107</li> 
+                <li class="search-bar-txt">days</li>
+                <li><button class="join-btn"> join us</button></li>
+                <li class="menu-icon right-icon"><a href="#"><i class="fas fa-bars"></i></a></li>
+            </ul>
+        </nav>
+        <div class="fixed-header"><a href="#"></a> Did you know we have a TECH FESTIVAL? Check it out →</div>
+    </header>
+    <main>
+        <section class="first-section">
+            <div class="large-left gradient-img">
```

Comment:
- [ ] All the sections inside your main section have a lot of space between them. You should use the padding top or bottom (not both) to make the space smaller